static/src/javascripts/projects/commercial/modules/article-aside-adverts.js

```diff
@@ -3,14 +3,20 @@ import $ from 'lib/$';
 import config from 'lib/config';
 import mediator from 'lib/mediator';
 import fastdom from 'lib/fastdom-promise';
-
 import type { bonzo } from 'bonzo';
 
 const minArticleHeight: number = 1300;
-const minImmersiveArticleHeight: number = 600;
 
-const minContentHeight = (): number =>
-    config.page.isImmersive ? minImmersiveArticleHeight : minArticleHeight;
+const calculateAllowedAdSlots = (availableSpace: number): string => {
+    if (availableSpace > 600) {
+        return '1,1|2,2|300,250|300,274|300,600|fluid';
+    } else if (availableSpace > 274) {
+        return '1,1|2,2|300,250|300,274';
+    } else if (availableSpace > 250) {
+        return '1,1|2,2|300,250';
+    }
+    return '1,1|2,2';
```

Comment:
What is a 2x2?

Reply:
Good question! It was there before so I've left it in, but I have no idea whether that would ever appear in practice...